### ISSUE TITLE
Info tooltip with large texts

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis-ui#1155]
I wonder if we could place the text from the tooltip somewhere on the screen - perhaps make the field smaller and place the text along side of it?

![syndesis - mozilla firefox_305](https://user-images.githubusercontent.com/1306050/32171641-1b4b7ffe-bd79-11e7-87cb-c3f5998c8ea1.png)

**Comment by gashcrumb** [comment]
I would suggest cutting down the amount of text really, these tooltips are only meant for short descriptions to jog the user's memory, or make them go look at the documentation if they need more information.

**Comment by kahboom** [comment]
I have to agree with @gashcrumb, especially since there is no way to collapse/expand text like with a popover. Trunking the text would just defeat the purpose imo.
